docs/config/llms-txt-options.ts

```
@@ -0,0 +1,39 @@
+    return undefined;
+  });
+};
+
+// Generates llms.txt and llms-full.txt alongside the HTML build so AI
+// assistants can ingest the docs.
+export const llmsTxtOptions: PluginOptions = {
+  siteTitle: 'Axelix Documentation',
+  siteDescription:
+    'Axelix helps developers and QA engineers debug, test, and monitor Spring Boot Java/Kotlin applications.',
+  content: {
+    enableMarkdownFiles: true,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)